atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraVerifier.java

```diff
@@ -205,8 +205,7 @@ private static boolean attemptToCreateIfNotExists(CassandraKeyValueServiceConfig
     // swallows the expected TException subtype NotFoundException, throws connection problem related ones
     private static boolean keyspaceAlreadyExists(InetSocketAddress host, CassandraKeyValueServiceConfig config)
             throws TException {
-        try {
-            CassandraClient client = CassandraClientFactory.getClientInternal(host, config);
+        try (CassandraClient client = CassandraClientFactory.getClientInternal(host, config)) {
```

Comment:
Given we do similar checks in e.g. `sanityCheckRingConsistency()`, and closing the client involves resetting the input and output protocols, this seems right.